index.js

```diff
@@ -125,12 +125,27 @@ module.exports = function (message, options) {
       }, maxLength);
 
       if (index === 0) {
+        /**
+         * Need to adjust the top position of the speech bubble depending on the
+         * amount of lines of the message.
+         */
         if (array.length === 2) {
           topOffset -= 1;
         }
         if (array.length >= 3) {
           topOffset -= 2;
         }
+        /**
+         * The speech bubble will overflow the Yeoman character id the message
+         * has more than 7 lines. So we vertically center the bubble by adding
+         * empty lines on top of the greeting.
+         */
+        if (array.length > 7) {
+          for (var i = 0; i < Math.floor((array.length - 7) / 2); i++) {
+            greeting.splice(0, 0, '');
+          }
+          frame.top = pad.left(frame.top, 32);
```

Comment:
I end up with `45 ` after fixing the issue you spotted:

```javascript
frame.top = pad.left(frame.top, 45);
```
Which is:
28 characters `╭──────────────────────────╮`
17 charatcter  The yeoman character »columns« `    /___A___\   /`
-------------------------------------------------------------------------------------------------------------
45 character


What do you mean by making it constant?